core/src/impl/Kokkos_ViewMapping.hpp

```diff
@@ -647,32 +647,37 @@ struct ViewOffset<
            m_dim.N5 * m_dim.N6;
   }
 
-  // Stride with [ rank ] value is the total length
   template <typename iType>
-  KOKKOS_INLINE_FUNCTION void stride(iType* const s) const {
+  KOKKOS_INLINE_FUNCTION void stride_fill(iType* const s) const {
```

Comment:
Adding a comment saying what `stride_fill` does and how it relates to `stride` might help navigate this code.

Reply:
I'll add one for stride too as even though it's internal-only it's susceptible to array overrun bugs

Reply:
Done, and fixed the bug in the code...